tests/scripts/run_performance.sh

```diff
@@ -51,6 +51,8 @@ run_perf_models_cnn_javelin() {
 
     if [ "$tt_arch" == "wormhole_b0" ]; then
         env WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/experimental/functional_stable_diffusion/tests -m $test_marker
+
```

Comment:
Do you mind putting `@skip_for_grayskull` directives in both these tests and getting rid of the `if tt_arch` check?